src/main/drivers/serial_uart.h

```diff
@@ -37,7 +37,8 @@ typedef enum {
     UARTDEV_5 = 4,
     UARTDEV_6 = 5,
     UARTDEV_7 = 6,
-    UARTDEV_8 = 7
+    UARTDEV_8 = 7,
+    UARTDEV_9 = 8
```

Comment:
Adding a comma after this element will result in a neater `diff` the next time this is appended.

Reply:
Well, `UART9` is the last UART we can add this way, as `serial_xx 10` is soft-serial lol

I changed it anyways, just in case.

Reply:
Ah, soft serial starts at 11 !!!